### DISCUSSION TITLE
How to get my account verified as a Student

### DISCUSSION THREAD

**Discussion by AJBrownson** [community/community#51074] [GitHub Education]
My question is not entirely unique but I just need to ask
I've been trying to upload my School ID card to get approved for the Student Developer Pack but to no avail
My uploaded photo is as clear as it can be and my name, institution's name and date on the ID card is legible and I don't know what else to do
Let the GitHub team please look into this issue and offer solutions
Thanks
![image](https://user-images.githubusercontent.com/92381518/227808886-9356ddb5-5a7c-4750-8ed9-b194651313dd.png)

**Comment by Ntshangase** [comment]
The verification process cross references your GitHub username, email and profile name (and other aspects). Do any of these match on your evidence?

Although Dated School ID is preferred GitHub does support alternative evidence.

**Comment by samus-aran** [comment]
Hi @AJBrownson 👋 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

**Comment by randyb100** [comment]
I'm going through the same thing and the webcam function is broken. It will only let me take photos with the selfie cam. I took a photo of my portal, a letter from the National Student Clearinghouse verifying my current enrollment, and of a financial aid statement that includes my name, address, and student ID number. None of it was accepted. I'm using my student email address. I don't know how to validate that I'm a remote learner since none of the documents say that, and I don't know of any college that would say that on their documents. My university is almost solely online.

**Reply by EHuffman1706** [reply]
So am I. I also can't or rather I don't have the option to verify via my school email address.

**Comment by nathanielborowski-ops** [comment]
I’m running into a similar issue with verification.

I have a valid student email address, which I used to create my GitHub account, and I also have multiple forms of documentation—proof of enrollment, scholarship documents, tuition records, and grades. All of these clearly match the name on my GitHub profile.

The only limitation is that I don’t have physical printed copies, so I’ve been pulling up the documents on my laptop and taking photos for submission. I’m starting to think that might be the reason my application isn’t being approved.

Is there a preferred way to submit digital documents? I’m a bit surprised there isn’t a clear option to upload files directly, unless I’m overlooking something.

Any insight would be appreciated.
